### PR TITLE
[BUG] Fix tutorial from Epochs Concatenation

### PR DESCRIPTION
**Pull request by adam2392**
#### Reference issue
Fixes: #10150

### CODE REVIEW

**Review by agramfort — COMMENTED** [review]
can you try to fix this in this PR?

I don't like having such a warnings catching in user doc

**Comment by larsoner** [comment]
I think fixing it might be a lot of work. I'm the dev meeting we talked about it not being a blocker for 1.0

Rather than catch, you could make sure concatenate_epochs has a verbose kwarg, then just pass verbose='error', and add a one line comment saying that this ignores the warning about losing annotations

**Comment by agramfort** [comment]
fine with me

**Comment by drammock** [comment]
for me a sensible fix is adding a partial `epochs.set_annotations()` implementation now, that only supports `None` (so the annotations can be removed before concatenating).  Then the only change to the tutorial is one line just before `concatenate_epochs` that says

```py
epochs.set_annotations(None)  # avoid warning about concatenating w/ annotations
```

That will get our docs building again, and I think @adam2392 was intending to flesh out the `set_annotations` functionality this week anyway (?) so it won't be a partial implementation for very long

**Comment by adam2392** [comment]
> for me a sensible fix is adding a partial `epochs.set_annotations()` implementation now, that only supports `None` (so the annotations can be removed before concatenating). Then the only change to the tutorial is one line just before `concatenate_epochs` that says
> 
> ```python
> epochs.set_annotations(None)  # avoid warning about concatenating w/ annotations
> ```
> 
> That will get our docs building again, and I think @adam2392 was intending to flesh out the `set_annotations` functionality this week anyway (?) so it won't be a partial implementation for very long

Done. Hopefully this passes :p.

**Comment by adam2392** [comment]
Not sure why CI is failing, but everything LGTM

File: tutorials/io/60_ctf_bst_auditory.py [hunk 1]
```diff
@@ -34,6 +34,7 @@
 import os.path as op
 import pandas as pd
 import numpy as np
+import warnings
```
Comment (larsoner) [inline tutorials/io/60_ctf_bst_auditory.py:37]:
```suggestion
```

**Comment by larsoner** [comment]
Thanks @adam2392 !

**Pull request merged**